server/handler/issue_comment.go

```diff
@@ -88,20 +88,33 @@ func (h *IssueComment) Handle(ctx context.Context, eventType, deliveryID string,
 	}
 
 	if fetchedConfig.Valid() {
-		tampered, err := h.detectAndLogTampering(ctx, prctx, client, event, fetchedConfig.Config)
-		if err != nil {
-			return errors.Wrap(err, "failed to detect tampering")
```

Comment:
I believe this would only catch the case where posting the updated failure status failed, since every other return provides `nil` for `err`. Therefore, I don't think this degrades this check. 

If we want to keep this error, PostStatus can be changed back to returning errors, and they can be processed by the callers of PostStatus.

Reply:
Any log monitoring should use the audit event, which I believe will always happen. This change sounds fine to me.